scan/context.go

```diff
@@ -88,35 +87,40 @@ func (s *Scanner) getContextFromGitURL(gitURL string) (contextDir string, err er
 }
 
 func (s *Scanner) getContextFromURL(remoteURL string) (err error) {
-	response, err := getWithStatusError(remoteURL)
-	if err != nil {
-		return errors.Wrapf(err, "unable to download remote context from %s", remoteURL)
```

Comment:
Could the URL be logged? Seems it contains SAS.

Reply:
It's very risky to log full URL as it can contains sensitive information.